### PR TITLE
Feature/meaningful meetings

### PR DESCRIPTION
**Pull request by ridlees**

### CODE REVIEW

**Comment by ridlees** [comment]
The new Warning system is really cool 👍🏻

File: practices/meaningful-meetings.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+- meeting agenda
+- meetings roles
+- organising meeting
+- events
+- moderator in meetings
+covered_by_dx_scanner: false
+---
+
+# Meaningful Meetings
+**TL;DR** Meaningful Meetings are controlled and designed meetings with a set of practices. These practices aim to minimise time spent on the meeting while maximising its outcome.
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:25]:
```suggestion

**TL;DR**

Meaningful Meetings are controlled and designed meetings with a set of practices. These practices aim to minimise time spent on the meeting while maximising its outcome. 
```

File: practices/meaningful-meetings.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+## How to Implement Meaningful Meetings?
+
+Committing to making meetings meaningful is a long run but even small practices can make the situation better. These practices can even improve your [Agile Events](https://developerexperience.io/practices/agile-events). Agile Events  are in their design meaningful.
+
+### Start With Reasons for Your Meeting
+
+An important practice of Meaningful Meetings is to always have a good reason for meetings. If you are unsure why you should have a meeting, dismiss it. 
+
+Common Bad Reasons for meetings are:
+- To update on information
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:60]:
```suggestion

- To update on information 
```

File: practices/meaningful-meetings.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+#### Add Outcomes to Your Agenda
+
+A meaningful meeting has trackable outcomes. Add the expected outcomes to your Agenda to track how efficient the meeting was. _Expected outcomes answer your Meeting Reasons._ Otherwise, your meeting is set up poorly.
+
+### Invite Relevant People
+
+With thinking that a meeting minute is a wasted minute, you start looking at people present on meetings differently. They can be doing different things right now, that directly translate to team success. 
+
+An interesting Framework to know who should be present is the RAP(I)D Framework. The I person is optional.
+- The person who **R**ecommends a decision or action
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:85]:
```suggestion

- The person who **R**ecommends a decision or action
```

File: practices/meaningful-meetings.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+### Design Your Meetings for Your Team 
+Designing your meetings means creating a structure of meetings that works best for your team. Consider adjusting the:
+- Length of meetings
+- Location of meetings
+- Policies of meetings
+- Roles present on meetings
+
+#### Interesting Meeting Policies to Consider:
+##### No Phones on Meetings
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:101]:
```suggestion

##### No Phones on Meetings
```

**Review by prokopsimek — COMMENTED** [review]
Some formatting fixes according to https://preview.developerexperience.io/practices/meaningful-meetings

File: practices/meaningful-meetings.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+
+All these bad reasons are solvable using [Agile Events](https://developerexperience.io/practices/agile-events) and product management tools. They don’t need any meetings at all.
+
+Before you schedule a meeting, have a clear goal. Know precisely why you and other people should _waste_ their work time on the meeting.
+
+### Create Meeting Agenda
+Meeting Agenda is a powerful and overlooked tool that rapidly improves meetings. The common way of creating a Meeting Agenda is to write your Reasons for meeting as a TODO list. We are doing this meeting to do X, Y, and Z. Be [SMART](#) with your Agenda.
+
+#### Add Relevant Information to Your Agenda
+Your Agenda should contain all information that is necessary for the meeting. Add documents, code, image, or other important information. Each piece of relevant information shortens the Meeting and contributes to [good developer experience](https://developerexperience.io/practices/good-developer-experience).
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:76]:
```suggestion
Your Agenda should contain all information that is necessary for the meeting. Add documents, code, image, or other important information. Each piece of relevant information shortens the Meeting and contributes to [good developer experience](/practices/good-developer-experience). 
```

File: practices/meaningful-meetings.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+![Meaningful meetings example](/files/meaningful-meeting.png)
+[Meaningful meetings example](https://pixabay.com/photos/startup-meeting-brainstorming-594090/)
+## Problems Meaningful Meetings Helps to Solve
+
+* Never Ending Meetings
+* Disorganised Meetings
+* [Demotivated Team (My opinion doesn’t matter feeling)](https://developerexperience.io/problems/demotivated-team)
+* [Toxic culture](https://developerexperience.io/problems/toxic-team-culture)
+* [Meaningless work](https://developerexperience.io/problems/meaningless-work)
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:51]:
```suggestion
* [Demotivated Team (My opinion doesn’t matter feeling)](/problems/demotivated-team)
* [Toxic culture](/problems/toxic-team-culture)
* [Meaningless work](/problems/meaningless-work)
```

**Comment by ridlees** [comment]
Shouldn't the bot in Preview (See **the/a** preview of the article at https://preview.developerexperience.io/) give the link to the article? (https://preview.developerexperience.io/practices/meaningful-meetings) Vím, že si to můžu na preview najít, ale kdyby to tam šlo automaticky, byla by to bomba

File: practices/meaningful-meetings.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+* Increased Cost
+
+## How to Implement Meaningful Meetings?
+
+Committing to making meetings meaningful is a long run but even small practices can make the situation better. These practices can even improve your [Agile Events](https://developerexperience.io/practices/agile-events). Agile Events  are in their design meaningful.
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:56]:
```suggestion
Committing to making meetings meaningful is a long run but even small practices can make the situation better. These practices can even improve your [Agile Events](/practices/agile-events). Agile Events are in their design meaningful.
```

File: practices/meaningful-meetings.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+An important practice of Meaningful Meetings is to always have a good reason for meetings. If you are unsure why you should have a meeting, dismiss it. 
+
+Common Bad Reasons for meetings are:
+
+- To update on information 
+- Because we scheduled one and we always do it on Monday
+- For team leader to monitor the process
+
+All these bad reasons are solvable using [Agile Events](https://developerexperience.io/practices/agile-events) and product management tools. They don’t need any meetings at all.
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:68]:
```suggestion
All these bad reasons are solvable using [Agile Events](/practices/agile-events) and product management tools. They don’t need any meetings at all.
```

File: practices/meaningful-meetings.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+
+An interesting Framework to know who should be present is the RAP(I)D Framework. The I person is optional.
+
+- The person who **R**ecommends a decision or action
+- The person who **A**grees formally
+- The person who **P**erforms and has accountability for the work
+- The person who **I**nputs extra details or points of view that may or may not align
+- The person who **D**ecides on the action and commits the organization to action.
+
+You can use [Team Roles](https://developerexperience.io/practices/team-roles-definition) to select who needs an invitation.
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:96]:
```suggestion
You can use [Team Roles](/practices/team-roles-definition) to select who needs an invitation.  
```

File: practices/meaningful-meetings.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+- The person who **A**grees formally
+- The person who **P**erforms and has accountability for the work
+- The person who **I**nputs extra details or points of view that may or may not align
+- The person who **D**ecides on the action and commits the organization to action.
+
+You can use [Team Roles](https://developerexperience.io/practices/team-roles-definition) to select who needs an invitation.  
+
+### Design Your Meetings for Your Team 
+Designing your meetings means creating a structure of meetings that works best for your team. Consider adjusting the:
+- Length of meetings
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:100]:
```suggestion

- Length of meetings
```

File: practices/meaningful-meetings.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+#### Interesting Meeting Policies to Consider:
+
+##### No Phones on Meetings
+The idea of no phones present on meetings as policy is to minimise distractions. Without distractions, you can finish your meeting agenda quicker. 
+
+##### 6-page Narratives
+The 6-page narrative is a rule of the famous Jeff Bezos and his meeting policy. In its simplest way, each Amazon meeting starts with reading 6-page full sentences memo and they absorb the information before talking. The memos are like dissertation work, structured about questions (reasons for the meeting), and all possible information around them. Writing in full sentences forces meeting members to debate with themselves. Asking for sources (data) makes your arguments stronger.  
+
+If you want to try the 6-page narrative (or 2-page, or X-page) for yourself, you can start with this [template](https://www.quora.com/Amazon-com-product/How-are-the-six-page-narratives-structured-in-Jeff-Bezos-S-Team-meetings):
+1. The issue or the question
```
Comment (prokopsimek) [inline practices/meaningful-meetings.md:114]:
```suggestion

1. The issue or the question
```

**Pull request merged**